scvi/inference/fish.py

```diff
@@ -65,8 +72,8 @@ def __init__(self, model, gene_dataset_seq, gene_dataset_fish, train_size=0.8, t
         self.train_seq, self.test_seq = self.train_test(self.model, gene_dataset_seq, train_size, test_size, seed)
         self.train_fish, self.test_fish = self.train_test(self.model, gene_dataset_fish,
                                                           train_size, test_size, seed, FishPosterior)
-        self.test_seq.to_monitor = ['ll']
-        self.test_fish.to_monitor = ['ll']
+        self.test_seq.to_monitor = ['reconstruction_error']
+        self.test_fish.to_monitor = ['reconstruction_error']
```

Comment:
Do you want double quotes here? I prefer double quotes I guess but I don't mind a mix either. But other places it seems like you're switching to double.

Reply:
I care for consistency and prefer double quotes. As I was modifying posterior.py I thought i'd harmonize to double quotes. I did not do it here to avoid breaking potential consistency.